server/pages.js

```diff
@@ -54,6 +54,7 @@ const pages = routes()
   .add('orders', '/:parentCollectiveSlug?/:collectiveType(events)?/:collectiveSlug/orders')
   .add('order', '/:parentCollectiveSlug?/:collectiveType(events)?/:collectiveSlug/orders/:OrderId([0-9]+)')
   .add('confirmOrder', '/orders/:id([0-9]+)/confirm')
+  .add('markOrderAsPaid', '/orders/:id([0-9]+)/markAsPaid')
```

Comment:
I think we prefer to enforce snake case for new URLs, @znarf?

```suggestion
  .add('markOrderAsPaid', '/orders/:id([0-9]+)/mark-as-maid')
```

Reply:
yes